pennylane/labs/vibrational_ham/localize_modes.py

```diff
@@ -0,0 +1,206 @@
+"""This module contains functions to localize normal modes."""
+
+import warnings
+import numpy as np
+import scipy
+
+au_to_cm = 219475
+hbar = 6.022 * 1.055e12  # (amu)*(angstrom^2/s)
+c_light = 3 * 10**8  # m/s
```

Comment:
```suggestion
AU_TO_CM = 219475
HBAR = 6.022 * 1.055e12  # (amu)*(angstrom^2/s)
C_LIGHT = 3 * 10**8  # m/s
```

Just convention for constants is to use all uppercase in python since python doesn't have like a `const` keyword other languages do ^_^.